Client/Pages/AuthorPages/AuthorView.razor

```diff
@@ -0,0 +1,125 @@
+﻿@page "/authorView"
+@using Microsoft.AspNetCore.Components.Forms
+@inject HttpClient Http
+@using Test.Shared
+
+<h3>AuthorView</h3>
+
+@if (authors.Count() != 0)
+{
+    <table class="table">
+        <tbody>
+            <tr>
+                <th>Author Name</th>
+                <th>Author Id</th>
+                <th>Author Books</th>
+                <th>Author born</th>
+                <th>Author died</th>
+            </tr>
+            @foreach (var author in authors)
+            {
+                <tr>
+                    <td><NavLink class="button" href=@($"/authorView/{author.AuthorId}")>@author.FullName</NavLink></td>
+                    <td>@author.AuthorId</td>
+                    <td>@foreach (var book in author.Books) 
+                    {
+                        <div><NavLink class="button" href=@($"/bookView/{book.BookId}")>@book.BookName</NavLink></div>
+                    }
+                    </td>
+                    <td>@author.YearBorn</td>
+                    <td>@author.YearDied</td>
+                    <td><button class="button" @onclick="(() => DeleteAuthor(author.AuthorId))">Delete author</button></td>
+                </tr>
+            }
+        </tbody>
+    </table>
+}
+<EditForm Model="_author" OnValidSubmit="@AddAuthor">
+    <DataAnnotationsValidator />
+    <ValidationSummary />
+    <table class="table">
+        <thead>
+            <tr>
+                <th scope="col">Author firstname</th>
+                <th scope="col">Author lastname</th>
+                <th scope="col">Year born</th>
+                <th scope="col">Year Died</th>
+                <th scopr="col">Books</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td><InputText placeholder="Author firstname" @bind-Value="_author.FirstName" /></td>
+                <td><InputText placeholder="Author lastname" @bind-Value="_author.LastName" /></td>
+                <td><InputNumber placeholder="Year born" @bind-Value="_author.YearBorn"></InputNumber></td>
+                <td><InputNumber placeholder="Year died" @bind-Value="_author.YearDied"></InputNumber></td>
+                <td>
+                    @foreach (var book in books)
+                    {
+                        <div><input type="checkbox" @onchange="EventArgs => CheckBoxClicked(book.BookId, EventArgs.Value)" />@book.BookName</div>
+                    }
+                </td>
+            </tr>
+        </tbody>
+    </table>
+    <button type="submit">Submit</button>
+</EditForm>
+
+@code {
+    private List<Author> authors { get; set; } = new List<Author>();
+    private List<Book> books { get; set; } = new List<Book>();
+    private Author _author { get; set; } = new Author();
+    private BookAuthor _bookAuthor { get; set; } = new BookAuthor();
+    public List<Guid> BooksIds { get; set; } = new List<Guid>();
```

Comment:
Side note: Private members should be prefixed with underscores. Unfortunately MS itself is not very clear and consistent about this. Their coding style states, that any form of Hungarian notation should not be used. However, this goes mostly for public or protected names. Private variables seem to be out of scope of any guidelines. Nevertheless, many IDE-s seem to whine if the private entities do not start with underscore. The reasoning behind this is that it avoids using `this` keyword as much as possible (e.g. when assigning value to private variable, it is possible to simply state `_bookAuthor = bookAuthor` in any function that takes `bookAuthor` as an input argument.